src/Resources/acme/ansible/group_vars/app.yml

```diff
@@ -0,0 +1,131 @@
+  mysql:          true
+  mysql_version:  '5.6'
+
+app_patterns:
+
+  ########
+  # Motd #
+  ########
+
+  motd_template: template/cow.j2
+  motd_message:  I can see clearly now - Jimmy Cliff
+
+  ############
+  # Timezone #
+  ############
+
+  timezone_default: Etc/UTC
+
+  ###############
+  # Environment #
+  ###############
+
+  environment_variables:
+    - SYMFONY_ENV:                 "{{ app.env }}"
+    - APP_SECRET:                  ThisTokenIsNotSoSecretChangeIt
+    - APP_DATABASE_HOST:           127.0.0.1
+    - APP_DATABASE_PORT:           null
+    - APP_DATABASE_NAME:           app
+    - APP_DATABASE_USER:           "{{ app.user }}"
+    - APP_DATABASE_PASSWORD:       null
+    - APP_DATABASE_SERVER_VERSION: 5.6
+    - APP_DOMAIN:                  "{{ app.host }}"
+
+  #########
+  # Files #
+  #########
+
+  files_attributes:
+    - path:  "{{ app.dir }}{{ app.dir_release }}/var/logs"
+      src:   "{{ app.log_dir }}"
+      state: link_directory
+    - path:  "{{ app.dir }}{{ app.dir_release }}/var/cache"
+      src:   "{{ app.cache_dir }}"
+      state: link_directory
+
+  #######
+  # Npm #
+  #######
+
+  npm_packages:
+    - package: gulp
+      version: 3
+
+  #######
+  # Php #
+  #######
+
+  php_extensions:
+    # Symfony
+    - intl
+    - curl
+    - mbstring
+    - xml
+    # Composer
+    - zip
+    # App
+    - gd
+
+  php_configs:
+    - file: app_opcache.ini
+      template: configs/app_opcache.{{ env }}.j2
+    - file: app.ini
+      template: configs/app.{{ env }}.j2
+      config:
+        - date.timezone: UTC
+        # App
+        - upload_max_filesize: 100M
+        - post_max_size:       100M
+
+  #########
+  # Nginx #
+  #########
+
+  nginx_configs:
+    # Php fpm
+    - file:     app_php_fpm
+      template: configs/app_php_fpm.{{ env }}.j2
+    # Gzip
+    - file:     app_gzip
+      template: configs/app_gzip.{{ env }}.j2
+    # App
+    - file: app.conf
+      config:
+        - server:
+          - server_name: "{{ app.host }}"
+          #- server_name: "*.ngrok.io"
+          - root:        "{{ app.dir }}{{ app.dir_release }}/web"
+          - access_log:  "{{ app.log_dir }}/nginx.access.log"
+          - error_log:   "{{ app.log_dir }}/nginx.error.log"
+          - include:     conf.d/app_gzip
+          - location /:
+            - try_files: $uri /app.php$is_args$args
+          - location ~ ^/(app(_[-\w]+)?)\.php(/|$):
+            - include: conf.d/app_php_fpm
+            #- internal;
+          - location ~ ^/filemanager/(dialog|execute|upload|ajax_calls|force_download)\.php(/|$):
+            - include: conf.d/app_php_fpm
+            - client_max_body_size: 100M
+
+  ##############
+  # Supervisor #
+  ##############
+
+  supervisor_configs:
+    - file:     app.conf
+      template: configs/app_program.{{ env }}.j2
+      config:
+        - workflow-worker:
+          - command:        php bin/console app:workflow:worker --run-max=1000 --shutdown-on-exception --no-interaction -vv
```

Comment:
Damn... This comes from dry*a, and not only this part is custom, meaning that the real template contains the same 😓, I have to fix it after merging this one